packages/terra-form-select/src/_Option.jsx

```diff
@@ -68,17 +68,23 @@ const Option = ({
   ]);
 
   return (
+    // Voiceover is having issues in a multiselect dropdown where once it reads the first
```

Comment:
Is this resolved or is this still an issue? If it is still an issue, we should log a GitHub issue for this and reference it in these comments.